src/handlers.js

```diff
@@ -153,12 +180,13 @@ class Handlers {
 module.exports = Handlers;
 
 /**
- * Post updates to GitHub, when the status of a task changes.
- * Taskcluster States: https://docs.taskcluster.net/reference/platform/queue/references/events
- * GitHub Statuses: https://developer.github.com/v3/repos/statuses/
- **/
-async function statusHandler(message) {
-  let taskGroupId = message.payload.taskGroupId || message.payload.status.taskGroupId;
+ * Modify the build in the Azure table when the taskgroup is resolved
+ *
+ * @param exchange message from Queue service
+ * @returns {Promise<void>}
+ */
+async function groupStatusHandler(message) {
```

Comment:
Is it even necessary to listen to task statuses anymore?  I suspect we could instead poll the github API for the checkSuite for the given repo / sha, and not even need the Builds table anymore.

I think keeping one kind of status in the Builds table and one kind in the checkSuite's will cause confusion, where the last commit has a green checkmark (checkSuite OK) but the SVG icon says the build failed (taskgroup is not 100% success).

Reply:
you mean group statuses? I wasn't sure because I am best familiar with tc-gh, and not as much with other services, so I don't know how Builds table fits into the overall picture

`CheckRuns` table doesn't keep statuses - it keeps check run IDs, so that we could update the statuses later

Reply:
Or are you talking about the table you proposed in the channel? Since we have all the check run ids, we don't need to store the statuses - we can ask github

Reply:
I think maybe we can even get rid of the builds table and anything to do with task group status! As you two were discussing in irc, just having a mapping of task -> check makes sense. The `taskgroupresolved` stuff has always been kinda flaky/confusing anyway.

We can probably avoid having a flag by proactively finding which orgs have tc installed and asking them to update our permissions way before we actually make the switch to the checks api. Ideally we don't actually require any changes to the tasks themselves by default?

This would also simplify the handler code drastically!

Reply:
Yes, sorry, I meant "group statuses" in the first comment in this thread, sorry for the confusion.